README.md

```diff
@@ -78,114 +78,97 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
+- Fixed bug where changing RGB values manually doesn't automatically update the color displayed. Thanks @martinchrzan!
 
 ### FancyZones
 
-- Adjusted "Save and apply" editor button to adjust with text size for localizations. Thanks @niels9001!
-- Fixed "Create new layout" button visibility when in high contrast mode. Thanks @niels9001!
-- Fixed scaling quirks related to editor UI. Thanks @niels9001!
-- Fixed editor crashing when double clicking the "edit layout" button.
-- Fixed issue with editor crashing immediately after displaying zones.
-- Fixed bug when navigating editor options via keyboard where pressing enter on unselected Canvas option launches Grid editor instead.
-- Fixed issue where FancyZones would not restore Console Applications.
-- Fixed Canvas editor and Grid editor window heights. Thanks @niels9001!
-- Fixed crash due to KERNELBASE.dll.
-- Fixed FancyZone icons to be smoother at higher DPI settings. Thanks @niels9001!
-- Fixed crash when changing between zone layouts.
-- Fixed regression where FancyZones does not resize windows on layout change.
-- Adjusted layout settings to reset shortcut key after canceling changes on a particular layout.
+- Fixed regression where restarting computer resets user defined layouts to the default selection.
+- Fixed issues with Grid layout editor not showing the "Save" and "Cancel" buttons.
+- Fixed accessibility issue where users could not add or merge zones using the keyboard.
+- Added a flyout describe the prerequisites for the "Allow zones to span across monitors" option. 
+- Fixed various crashing bugs.
 
 ### File Explorer add-ons
 
-- Fixed issue where markdown files were still previewed even when "Enable Markdown" was turned off.
+- Added PDF Thumbnail Provider for Windows Explorer. Thanks @rdeveen!
 
 ### Image Resizer
 
-- Added warning that GIF files with animations may not correctly resize if the encoding used for the files is incompatible.
+- Added default values for newly added sizes. Thanks @htcfreek!
+- Fixed regression where spaces in the filename format settings couldn't be registered.
+- Corrected scaling issues with Image Resizer Window. Thanks @niels9001!
+- Fixed issue where PowerToys crashes when json settings are not formatted properly. Thanks @davidegiacometti!
 
 ### Keyboard Manager
 
-- Improved UI for KBM re-mappings list. Thanks @niels9001!
+- Fixed crash when adding a shortcut.
+- Fixed issue with Re-mappings window not displaying.
+- Fixed issue when remapping a shortcut to `Alt`+`Tab` breaks the `Alt`+`Tab` navigation with arrow keys.
 
-### PowerRename
+### PowerToys Run
 
-- Expanding a plugin option in settings can now be toggled. Thanks @niels9001!
-- Fixed race condition causing PowerRename to crash File Explorer. Thanks @ianjoneill!
+- Added path filters for Settings plugin via `>` key. Thanks @htcfreek! 
+- Translation improvements for Settings plugin. Thanks @htcfreek! 
+- Fixed exception errors for Settings Plugin. Thanks @htcfreek!
```

Comment:
This wasn't user facing and only on dev build. It was introduced by a code change I made before. - Maybe nothing for release notes.